aim/api/resource.py

```diff
@@ -66,6 +85,8 @@ class BridgeDomain(ResourceBase):
     # Attrbutes completely managed by the DB (eg. timestamps)
     db_attributes = []
 
+    _aci_mo_name = 'fvBD'
```

Comment:
Honestly speaking we can't paint the whole picture for debugging anyways because there are so many things in ACI that are beyond AIM control. Instead we should strive to provide concise meaningful information that will *enable* the end user to debug ACI.

Regarding ugliness: in the case of fvBD, fvRsCtx is actually the ugly part. So it should not be part of the AIM resource because we don't want AIM clients to be bothered about it. fvRsCtx exists only in AID right now, and that is great. Exposing the DN of fvRsCtx through AIM will breach the abstraction that AIM is trying to provide, no?